src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-cell;
+  width: $with-calendar;
```

Comment:
There is a typo in the variable name. It should be $width-calendar instead of $with-calendar.